scripts/run-dependencies

```diff
@@ -9,9 +9,59 @@ HERE=$(realpath "$(dirname $0)")
 
 docker exec packit-db wait-for-db
 
-# Start outpack_server
-docker pull $OUTPACK_SERVER_TAG
-docker run --name outpack_server \
-  -v $HERE/../demos/orderly:/outpack \
-  -p 8000:8000 -d --rm \
-  $OUTPACK_SERVER_TAG
+docker network create $NETWORK
```

Comment:
could put all this into a orderly.runner script that gets called... can then also take all env vars out of common and use thre also

Reply:
can we use just the default bridge network?

Reply:
i would prefer to keep it in the run dependencies script, i think its okay if theres a couple more containers being run in the same script, i think its more annoying and time consuming to have lots of referencing of other scripts to work out whats going on

default bridge network doesnt work, the server and workers cannot find the redis connection, i think it has to be a separate network